modules/cloudfront-proxy-config/variables.tf

```diff
@@ -15,6 +15,34 @@ variable "proxy_config_version" {
   }
 }
 
+################
+# DynamoDB Table
+################
+
+variable "multiple_deployments" {
+  description = "Have multiple deployments and domain aliases."
+  type        = bool
+  default     = false
+}
+
+variable "proxy_config_table_name" {
+  description = "Name of the DynamoDB table to store proxy configurations."
+  type        = string
+  default     = "tf-next-proxy-config"
+}
+
+variable "proxy_config_table_write_capacity" {
+  description = "Capacity for writing to the proxy configuration DynamoDB table."
+  type        = number
+  default     = 5
```

Comment:
When adding a preconfigured capacity here, would that cause upfront payments (instead of on-demand pricing) in AWS?
Since capacity is marked as optional in the docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/dynamodb_table

Not against adding the provisioned capacity but for quick testing of the module we should make on-demand pricing the default choice.
Provisioned capacity could then be configured via input variables if needed.

Reply:
Okay, I made "pay per request" the default now. Let's add configurability later.